02/app.js

```diff
@@ -1,11 +1,19 @@
 const calendarJS = {
-    'ES1': '1997-06',
-    'ES2': '1998-06',
-    'ES3': '1999-12',
-    'ES4': null,
-    'ES5': '2009-12',
-    'ES6': '2015-06',
-    'ES7': '2016-06',
-    'ES8': '2017-06',
-    'ES9': '2018-06',
-}
\ No newline at end of file
+  ES1: "1997-06",
+  ES2: "1998-06",
+  ES3: "1999-12",
+  ES4: null,
+  ES5: "2009-12",
+  ES6: "2015-06",
+  ES7: "2016-06",
+  ES8: "2017-06",
+  ES9: "2018-06",
+};
+
+for (const key in calendarJS) {
+  if (calendarJS[key]) {
```

Comment:
Jest ok, ale trzeba pamiętać, że jeśli chcielibyśmy sprawdzać czy jest to `null` to trzeba jawnie napisać. 
Inaczej każda wartość fałszywa zadziała tak jak `null`.